README.md

```diff
@@ -166,18 +166,18 @@ an excelent toolkit for information retrieval research.
 First, we need to install Maven, and clone and compile Anserini's repository:
 ```
 sudo apt-get install maven
-git clone https://github.com/castorini/Anserini.git
+git clone --recurse-submodules https://github.com/castorini/Anserini.git
 cd Anserini
 mvn clean package appassembler:assemble
-tar xvfz eval/trec_eval.9.0.4.tar.gz -C eval/ && cd eval/trec_eval.9.0.4 && make
+tar xvfz tools/eval/trec_eval.9.0.4.tar.gz -C tools/eval/ && cd tools/eval/trec_eval.9.0.4 && make
```

Comment:
the eval and ndeval folder are now found inside the tools directory ( this directory is a clone of the anserini-tools rep)